docs/table/construct_table.rst

```diff
@@ -14,8 +14,8 @@ Examples
 Much of the flexibility lies in the types of data structures
 which can be used to initialize the table data. The examples below show how to
 create a table from scratch with no initial data, as well as create a table
-with a list of columns, a dictionary of columns, or from ``numpy`` arrays
-(either structured or homogeneous).
+with a list of columns, a dictionary of columns, from ``numpy`` arrays (either
+structured or homogeneous), or from a |SkyCoord| object.
```

Comment:
The docs here refer to data types that can be passed to the `Table` constructor as the first argument. So you should revert this change here since you cannot do `Table(SkyCoord(...))`.

Reply:
There are sections about converting a `SkyCoord` and a `pandas.DataFrame` in the text below, so they should also be mentioned here, but it would indeed be better to clearly distinguish between what can be directly passed to the `Table` constructor and what can be converted with separate functions.

Reply:
I see the first sentence of the `Examples` section is redundant from above anyway. Let's just make this bit simpler with:
```
Examples
=======

The examples below demonstrate a variety of ways to create a |Table| or |QTable|,
either by using the table class constructor or by converting other objects to a table.
```

Reply:
The preceding section already says that examples will follow, and whether creating a table involves the constructor explicitly or is done through a separate function is obvious from the relevant example. So it seems to me this new proposed sentence is not really needed either and the examples can simply start with Setup.